CMakeLists.txt

```diff
@@ -337,18 +336,25 @@ ELSE(CMAKE_TOOLCHAIN_FILE)
     SET(MPIEXEC_NUMPROC_FLAG "-n")
 
     IF(QMC_CUDA)
+      IF(QMC_BUILD_STATIC)
+        MESSAGE(FATAL_ERROR "Static linking is prohibited with CUDA build!")
```

Comment:
Change to "Static linking is not supported with CUDA build. Set QMC_BUILD_STATIC=0"